src/App.vue

```diff
@@ -498,7 +500,7 @@ export default class App extends Mixins(
   }
 
   /** Called when component is destroyed. */
-  private destroyed (): void {
+  protected beforeDestroy (): void {
```

Comment:
This is the new lifecycle event name in Vue3 but it's back-compatible with Vue2.